actionview/lib/action_view/template/handlers/erb/erubi.rb

```diff
@@ -18,6 +18,9 @@ def initialize(input, properties = {})
             properties[:preamble]   ||= ""
             properties[:postamble]  ||= "#{properties[:bufvar]}.to_s"
 
+            # Tell Eruby that whether template will be compiled with `frozen_string_literal: true`
+            properties[:freeze_template_literals] = !Template.frozen_string_literal
```

Comment:
I'm not super happy with having to reach to template here, but since the `erb_implementation` could be anything we can't pass arbitrary options :/

Does allowing to change `erb_implementation` really have value? Can't people replace the entire handler ?